api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
```

Comment:
Rephrase: Developers can easily construct and efficiently execute data stream pipelines consisting of neural networks as filters in pipelines.

Reply:
it's applied